detox/android/detox/src/full/java/com/wix/detox/espresso/DetoxMatcher.java

```diff
@@ -47,6 +48,14 @@ public static Matcher<View> matcherForContentDescription(String contentDescripti
         return allOf(withContentDescription(contentDescription), withEffectiveVisibility(Visibility.VISIBLE));
     }
 
+    public static Matcher<View> matcherForContentDescriptionOrText(String contentDescription) {
+        return allOf(anyOf(withContentDescription(contentDescription), withText(contentDescription)), withEffectiveVisibility(Visibility.VISIBLE));
+    }
```

Comment:
I don't really get what you're doing here to be honest. Let's discuss this part..

Reply:
I'll try to improve conventions